packages/hooks/src/test/index.test.js

```diff
@@ -620,3 +628,30 @@ test( 'Test `this` context via composition', () => {
 	const testObject2 = {};
 	Object.assign( testObject2, createHooks() );
 } );
+
+// Test adding a hook triggers a hookAdded action.
+test( 'adding a hook triggers a hookAdded action passing all callback details', () => {
+	const testObject = {};
```

Comment:
`testObject` is never used in this test. The same applies in the line 648.

Reply:
right, fine to remove